.gitlab-ci.yml

```diff
@@ -293,15 +293,10 @@ integration:deployment:
       image=(./builds/*-docker-*);
       ./scripts/deploy-image.sh "${image[0]}" \'testnet\' "$CI_REGISTRY_IMAGE";
       '
-    - echo 'Deploying ECS service to testnet'
-    - >
-      nix-shell --run $'
-      ./scripts/deploy-service.sh \'polykey-testnet\';
-      '
     - echo 'Waiting for seed nodes to update'
     - >
       nix-shell --arg ci true --run $'
-      ./scripts/wait-for-deploy.js;
+      ./scripts/wait-for-deploy.js testnet;
```

Comment:
Maybe we could do this `./scripts/wait-for-deploy.js testnet.polykey.com`.

Reply:
Made the change and fixed it up in all 3 places.